src/test/java/com/placelab/utilis/WebDriverSetup.java

```diff
@@ -0,0 +1,37 @@
+package com.placelab.utilis;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.edge.EdgeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+
+public class WebDriverSetup {
+    public static WebDriver getWebDriver(final String browserName) {
+        switch (browserName.toUpperCase()) {
+            case "CHROME":
+                return chromeDriver();
+            case "FIREFOX":
+                return firefoxDriver();
+            case "EDGE":
+                return edgeDriver();
+            default:
+                throw new IllegalArgumentException("No match to the given browser name found!");
```

Comment:
when writing exception messages try to provide enough details in it, so for example if it happens that you provide unsupported browser name, the exception message includes the invalid string value that caused the exception

Reply:
Fixed. Thank you :)